src/wp-admin/security.php

```diff
@@ -0,0 +1,157 @@
+<span style="color: #BA2121; font-style: italic"> *</span>
+<span style="color: #BA2121; font-style: italic"> * This function takes a capability which will be used to determine whether</span>
+<span style="color: #BA2121; font-style: italic"> * or not a page is included in the menu.</span>
+<span style="color: #BA2121; font-style: italic"> *</span>
+<span style="color: #BA2121; font-style: italic"> * The function which is hooked in to handle the output of the page must check</span>
+<span style="color: #BA2121; font-style: italic"> * that the user has the required capability as well.</span>
+<span style="color: #BA2121; font-style: italic"> *</span>
+<span style="color: #BA2121; font-style: italic"> * @param string   $page_title The text to be displayed in the title tags of</span>
+<span style="color: #BA2121; font-style: italic">                               the page when the menu is selected.</span>
+<span style="color: #BA2121; font-style: italic"> * @param string   $menu_title The text to be used for the menu.</span>
+<span style="color: #BA2121; font-style: italic"> * @param string   $capability The capability required for this menu to be</span>
+<span style="color: #BA2121; font-style: italic">                               displayed to the user.</span>
+<span style="color: #BA2121; font-style: italic"> * @param string   $menu_slug  The slug name to refer to this menu by (should</span>
+<span style="color: #BA2121; font-style: italic">                               be unique for this menu).</span>
+<span style="color: #BA2121; font-style: italic"> * @param callable $function   The function to be called to output the content</span>
+<span style="color: #BA2121; font-style: italic">                               for this page.</span>
+<span style="color: #BA2121; font-style: italic"> * @return false|string The resulting page&#39;s hook_suffix, or false if the user</span>
+<span style="color: #BA2121; font-style: italic">                        does not have the capability required.</span>
+<span style="color: #BA2121; font-style: italic"> */</span>
+<span style="color: #008000; font-weight: bold">function</span> <span style="color: #0000FF">add_security_page</span>(
+    <span style="color: #19177C">$page_title</span>,
+    <span style="color: #19177C">$menu_title</span>,
+    <span style="color: #19177C">$capability</span>,
+    <span style="color: #19177C">$menu_slug</span>,
+    <span style="color: #19177C">$function</span> <span style="color: #666666">=</span> <span style="color: #BA2121">&#39;&#39;</span>
+)
+
+</pre>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="meta-box-sortables ui-sortable">
+          <div class="postbox">
+            <h2><span><?php _e( 'Usage' ); ?></span></h2>
+            <div class="inside">
+              <p>Lipsum</p>
+              <!-- HTML generated using hilite.me -->
```

Comment:
Same comment as above regarding `hilite.me` usage, though just saying "see above for details" should be sufficient here.